torch/utils/_foreach_utils.py

```diff
@@ -5,6 +5,19 @@
 from torch import Tensor
 from torch.autograd.grad_mode import no_grad
 
+__all__ = ["get_foreach_kernels_supported_devices", "get_fused_kernels_supported_devices"]
+
+def get_foreach_kernels_supported_devices() -> List[str]:
```

Comment:
Let's also keep these private by prefixing with underscore: _get_foreach_kernels_supported_devices and _get_fused_kernels_supported_devices